frontend/benefit/applicant/src/components/applications/forms/application/step1/companyInfo/CompanyInfo.component.tsx

```diff
@@ -0,0 +1,196 @@
+import { APPLICATION_FIELDS } from 'benefit/applicant/constants';
+import { TextInput } from 'hds-react';
+import React from 'react';
+import InputMask from 'react-input-mask';
+import { StyledCheckbox } from 'shared/components/forms/fields/styled';
+import FormSection from 'shared/components/forms/section/FormSection';
+
+import { useComponent } from '../extended';
+import SC from './CompanyInfo.sc';
+
+export interface CompanyInfoProps {
+  data: {
+    name: string;
+    streetAddress: string;
+    postcode: string;
+    city: string;
+    businessId: string;
+  };
+  error?: string;
+  loading?: boolean;
+}
+
+const CompanyInfo: React.FC<CompanyInfoProps> = ({ data, error, loading }) => {
+  const { t, getErrorMessage, fields, translationsBase, formik } =
+    useComponent();
+  let formattedData = { ...data, businessId: `Y-tunnus: ${data.businessId}` };
+  if (error)
+    formattedData = {
+      name: '-',
+      streetAddress: '-',
+      postcode: '-',
+      city: '',
+      businessId: '-',
+    };
+  if (loading)
+    formattedData = {
+      name: '',
+      streetAddress: '',
+      postcode: '',
+      city: '',
+      businessId: '',
+    };
+
+  return (
+    <FormSection header={t(`${translationsBase}.heading1`)} loading={loading}>
```

Comment:
why we cannot just use skeleton loader to show nicely loading animation instead of passing props and styles? I think the one we created in the events-helsinki project for the dynamic loaded section data pretty much suits the design